src/langchain_google_bigtable/document_loader.py

```diff
@@ -223,10 +264,25 @@ def add_documents(self, docs: List[Document]):
                 self.content_column_name,
                 doc.page_content.encode(self.content_encoding.value),
             )
+            unmapped_metadata = dict(doc.metadata)
+            unmapped_metadata.pop(ID_METADATA_KEY, None)
             for mapping in self.metadata_mappings:
                 if mapping.metadata_key in doc.metadata:
                     value = self._encode(doc.metadata[mapping.metadata_key], mapping)
                     row.set_cell(mapping.column_family, mapping.column_name, value)
+                    unmapped_metadata.pop(mapping.metadata_key)
+            if (
+                self.metadata_as_json_column_family is not None
+                and self.metadata_as_json_column_name is not None
```

Comment:
I'm concerned about one of these being none and the other just being ignored

Reply:
Fixed. If one is None and the other is not - we throw in the ctor.